### ISSUE TITLE
Missing chord 'Xdim'

### ISSUE THREAD

**Issue by scally1017** [trubadur-app/trubadur#6]
Missing diminished chords

**Comment by trubadur-app** [comment]
To be filled in by the first person who has the time:

```
Chord: dim

Guitar instructions:
A: 	[]
A#: 	[]
B: 	[]
C: 	[]
C#: 	[]
D: 	[]
D#: 	[]
E: 	[]
F: 	[]
F#:	[]
G: 	[]
G#: 	[]
```

**Comment by scally1017** [comment]
Chord: dim

Guitar instructions:
A: 	[X 0 1 2 1 X] , [5 6 7 5 X X] , [X X 7 5 4 5]
A#: 	[X 1 2 3 2 X] , [6 7 8 6 X X] , [X X 8 6 5 6]
B: 	[X 2 3 4 3 X] , [7 8 9 7 X X] , [X X 9 7 6 7]
C: 	[X 3 4 5 4 X] , [8 9 10 8 X X] , [X X 10 8 7 8]
C#: 	[X 4 5 6 5 X] , [9 10 11 9 X X] , [X X 11 9 8 9]
D: 	[X 5 6 7 6 X] , [10 11 12 10 X X] , [X X 12 10 9 10]
D#: 	[X 6 7 8 7 X] , [11 12 13 11 X X] , [X X 13 11 10 11]
E: 	[X 7 8 9 8 X] , [0 1 2 0 X X] , [X X 14 12 11 12]
F: 	[X 8 9 10 9 X] , [1 2 3 1 X X] , [X X 3 1 0 1]
F#:	[X 9 10 11 10 X] , [2 3 4 2 X X] , [X X 4 2 1 2]
G: 	[X 10 11 12 11 X] , [3 4 5 3 X X] , [X X 5 3 2 3]
G#: 	[X 11 12 13 12 X] , [4 5 6 4 X X] , [X X 6 4 3 4]

**Comment by trubadur-app** [comment]
Thanks for you help @scally1017 . This is now included in version 1.0.13.
